app/views/projects/_project_watch.html.haml

```diff
@@ -0,0 +1,31 @@
+#watch_buttons.btn-group.col-md-12
+  - if current_user.project_followings.include? project
+    = link_to unfollow_project_path(episode, project), class: 'btn btn-default watch_action' do
+      %i.fa.fa-eye-slash
+      Unwatch
+  - else
+    = link_to follow_project_path(episode ,project), class: 'btn btn-default watch_action' do
+      %i.fa.fa-eye
+      Watch
+  = link_to "#{project.project_followers.count}", '#', class: 'btn btn-default', data: { toggle: 'modal', target: '#watchersModal' }
+
+.modal.fade{id: "watchersModal","aria-labelledby" => "watchersModalLabel", role: "dialog", tabindex: "-1"}
```

Comment:
Ids and classes should be written as: `watchers-modal` and not `watchersModal` as in javacript.